src/octoprint/settings.py

```diff
@@ -0,0 +1,2390 @@
+"""
```

Comment:
This file was moved as part of the migration to a Pydantic model for the settings.

Make sure your branch is up to date with maintenance before creating a PR, to use the new settings models.